pkg/convert/xliff2.go

```diff
@@ -145,6 +140,116 @@ func ToXliff2(translation model.Translation) ([]byte, error) {
 
 // helpers
 
+// messageFromUnit converts Xliff2.0 unit element to model.Message.
+func messageFromUnit(u unit, original bool) (*model.Message, error) {
```

Comment:
what problem does this solve?

Reply:
// messageFromUnit converts Xliff2.0 unit element to model.Message.

Reply:
Examples?